ui/app/src/model/project-client.ts

```diff
@@ -37,3 +38,60 @@ export function useProjectQuery(options?: ProjectListOptions) {
     options
   );
 }
+
+/**
+ * Used to create a project in the API
+ *
+ * Will automatically invalidate the project list and force get query to be executed again.
```

Comment:
Nice, thank you

Reply:
You're welcome, I was pretty glad when I discovered that feature :D Otherwise I would have to find a way to share a state between the navbar and the view oO